mapping-for-navigation/_posts/0000-01-09-modeling-interesections-for-map-navigation.md

```diff
@@ -0,0 +1,191 @@
+---
```

Comment:
There’s a typo in the file name: “interesections” instead of “intersections”.

Reply:
fixed, 🙇